src/webui/www/private/css/dynamicTable.css

```diff
@@ -101,3 +101,42 @@ tr.dynamicTableHeader {
     padding-bottom: 0px !important;
     padding-top: 0px !important;
 }
+
+#transferList {
+    overflow: hidden !important; /* override for mocha inline styles */
+
+    & #transferList_pad {
+        display: flex !important;
+        flex-direction: column;
+        height: inherit;
+
+        & #torrentsTableDiv {
+            flex: 1;
+        }
+    }
+}
```

Comment:
I have a branch locally where I just solved this exact problem! But I switched all tables over to using flexbox. Here's the commit, if you're interested: https://github.com/Piccirello/qBittorrent/commit/69d66ac31d7c9fd718f510acde6ea34e9a679e67

Reply:
On second thought, how would you feel about me opening a PR and closing this current one? The CSS changes in mine are more generic and work for more tables while solving the same problems.

Reply:
Sure

Reply:
Opened #21652